frontend/src/features/admin-form/responses/ResponsesPage/storage/UnlockedResponses/ResponsesTable/ResponsesTable.tsx

```diff
@@ -34,12 +40,80 @@ const RESPONSE_TABLE_COLUMNS: Column<ResponseColumnData>[] = [
     Header: 'Timestamp',
     accessor: 'submissionTime',
     minWidth: 200,
-    width: 300,
+    width: 250,
     disableResizing: true,
   },
 ]
+const PAYMENT_COLUMNS: Column<ResponseColumnData>[] = [
+  {
+    Header: 'Email',
+    accessor: ({ payments }) => {
+      if (!payments?.email) {
```

Comment:
/nits this can be shorthanded to `return !payments?.email ? '' : payments.email`